### ISSUE TITLE
Idea: Option to hide the Featured section

### ISSUE THREAD

**Issue by dogweather** [thebaer/MMRA#18]
The first page of medium.com is the **Featured** section chosen by the editorial board, and there's no way to opt out. There's no way to signal disinterest in the articles, topics, or writers; as there is in the following sections, **Top stories for you**, **New from your network**, etc.

Are you open to this feature if I submit a PR?

**Comment by thebaer** [comment]
Absolutely! We should definitely be able to do something about that. I would only suggest that if you  wanted to hide it, it should be an option instead of an unchangeable behavior, as some people might want to stay opted in.
